.github/workflows/sponsors.yml

```diff
@@ -1,26 +0,0 @@
-name: Add Sponsor Labels
-
-on:
-  workflow_call:
-  pull_request:
-    types:
-      - opened
-  issues:
-    types:
-      - opened
-
-permissions:
-  issues: write
-  pull-requests: write
-  contents: read
-
-jobs:
-  sponsors:
-    name: Label Sponsors
-    runs-on: ubuntu-latest
-    steps:
-      - uses: JasonEtco/is-sponsor-label-action@v1.2.0
-        with:
-          label: Sponsor
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
was this file intended to be removed?

Reply:
yeah it's been broken for ages and I give up